Content/Items/Currency/EchoingShard.cs

```diff
@@ -0,0 +1,42 @@
+﻿using PathOfTerraria.Common.Enums;
+using PathOfTerraria.Core.Items;
+using Terraria.ID;
+
+namespace PathOfTerraria.Content.Items.Currency;
+
+internal class EchoingShard : CurrencyShard
+{
+	public override bool CanRightClick()
```

Comment:
This is mostly personal preference, but usually I make sure SetDefaults is at the top of the methods (preceded only by SetStaticDefaults and Load/IsLoadingEnabled). Means nothing and changes nothing but is a little more consistent and I thought I'd point it out.